src/main/java/moe/nightfall/vic/integratedcircuits/cp/CircuitProperties.java

```diff
@@ -64,6 +64,10 @@ public void addComment(Comment comment) {
 	public EnumConnectionType getModeAtSide(int side) {
 		return EnumConnectionType.values()[con >> (side * 2) & 3];
 	}
+	
+	public static EnumConnectionType getModeAtSide(int con, int side) {
+		return EnumConnectionType.values()[con >> (side * 2) & 3];
```

Comment:
Dont know what you need this for but call it from the other method at least (one above)